sway/config/output.c

```diff
@@ -393,6 +395,7 @@ bool apply_output_config(struct output_config *oc, struct sway_output *output) {
 		// output disabled for now and try again when the output gets the mode
 		// we asked for.
 		sway_log(SWAY_ERROR, "Failed to modeset output %s", wlr_output->name);
+		output->enabled = output->configured;
```

Comment:
Maybe it makes more sense to save the previous value of `output->enabled` prior to the `apply_output_config` call, and change it back here?

Reply:
Changed. Although I'm pretty sure it's a logic error if the value did not match output->configured.

Reply:
Yeah, the previous version worked. It just required the reader to understand that previously enabled outputs were always configured.